package.json

```
@@ -0,0 +1,13 @@
+{
+    "scripts": {
+      "lint": "eslint . --fix",
```

I'm not a fan of `--fix` here because I don't expect npm scripts to modify my source.

```suggestion
      "lint": "eslint .",
```